src/scss/shared/_base.scss

```diff
@@ -22,6 +25,14 @@
 			@include oTypographyMargin($bottom: 0);
 		}
 	}
+	
+	.o-forms-field--optional .o-forms-title--main:after {
```

Comment:
Is `.o-forms-title--main` a BEM modifier?

Reply:
well there's 
```
<span class="o-forms-title">
  <span class="o-forms-title--main">Input title</span>
  <span class="o-forms-title--prompt">prompt/description</span>
</span>
```

Reply:
Ah, it should be underscore for an "element".